assets/src/wizards/subscriptions/index.js

```diff
@@ -40,31 +44,65 @@ class SubscriptionsWizard extends Component {
 	 * Render the example stub.
```

Comment:
- right when the Wizard is loaded, updating the currentTask (since, for example, tasks 0 and 1 might already be done):

```suggestion
	 * componentDidMount.
	 */
	componentDidMount() {
		this.setState( { currentTask: this.getNextTask( 0, this.state.completedTasks) } )
	}
```

- just to keep things clean, extracted this into a separate function:

```suggestion

	/**
	 * Starting from the startingIndex, determine the next task to go to.
	 *
	 * @param int startingIndex
	 * @param Array completedTasks
	 * @returns int {number}
	 */
	getNextTask = ( startingIndex, completedTasks ) => {
		let nextTaskIndex = 0;
		while ( completedTasks.indexOf( nextTaskIndex ) !== -1 ) {
			nextTaskIndex++;
		}

		return nextTaskIndex;
	}
```

- here added a handler when the task is complete (besides updating the `currentTask`, now also adding it into the `completedTasks` Array)

```suggestion

	/**
	 * Upon completing the task, add its index to this.state.completedTasks, and increase this.state.currentTask.
	 *
	 * @param int taskIndex
	 */
	completeTask = ( taskIndex ) => {
		let completedTasks = [ ...this.state.completedTasks ];
		if ( completedTasks.indexOf( taskIndex ) === -1 ) {
			completedTasks.push( taskIndex );
			completedTasks.sort();

			this.setState( { completedTasks } );
		}

		this.setState( { currentTask: this.getNextTask( 0, completedTasks ) } );
	}

	/**
	 * Render the example stub.
```